### DISCUSSION TITLE
what kind of machine do I need？help me

### DISCUSSION THREAD

**Discussion by melowYX** [qdrant/qdrant#808] [General]
I have about 13GB of text data, which is about 100 million lines，and each row corresponds to a 384-dimensional vector.
If I am storing those data with qdrant ,what kind of machine do I need to configure the qdrant cluster？

**Comment by melowYX** [comment]
100 QPS

**Reply by generall** [reply]
Hi @melowYX you would need to use a distributed deployment option for this. A cluster of several machines.

**Reply by melowYX** [reply]
4vCPU 3.5 GHz，Memory 32GiB，How many machines do I need with this hardware configuration?

**Reply by generall** [reply]
about 6

**Reply by melowYX** [reply]
Thanks for your response
